Source/Core/Core/IOS/USB/Bluetooth/BTReal.cpp

```diff
@@ -373,7 +373,7 @@ void BluetoothReal::WaitForHCICommandComplete(const u16 opcode)
   for (int tries = 0; tries < 100; ++tries)
   {
     if (libusb_interrupt_transfer(m_handle, HCI_EVENT, buffer.data(),
-                                  static_cast<int>(buffer.size()), &actual_length, 20) == 0 &&
+                                  static_cast<int>(buffer.size()), &actual_length, TIMEOUT) == 0 &&
```

Comment:
Would it be best to set the timeout to TIMEOUT to allow for some "fast-responding" Bluetooth adapters to work while being shared over the network (with a total latency <200ms) and reduce the number of tries?
Or keep it to 20ms and loose the ability to share Bluetooth adapters over the network? (note that I haven't tested actually sharing a Bluetooth adapter over the network, but I assume it will fail like the GC adapter did)